x-pack/test/cases_api_integration/security_and_spaces/tests/common/comments/post_comment.ts

```diff
@@ -1123,6 +1123,36 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     });
 
+    describe('partial updates', () => {
+      it('should not result to a version conflict (409) when adding a comment to an updated case', async () => {
+        const postedCase = await createCase(supertest, postCaseReq);
+
+        /**
+         * Updating the status of the case will
+         * change the version of the case
+         */
+        await updateCase({
+          supertest,
+          params: {
+            cases: [
+              {
+                id: postedCase.id,
+                version: postedCase.version,
+                status: CaseStatuses['in-progress'],
+              },
+            ],
+          },
+        });
+
+        await createComment({
```

Comment:
As we always get the case before adding the comment (and thus updating the case) the test does not simulate a race condition. It is a basic test to verify the correctness of a simple path and to avoid breaking something in the future.